specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/examples/DataRegistries_GetDataRegistryInvalid.json

```diff
@@ -0,0 +1,36 @@
+{
+  "parameters": {
+    "geography": "us",
+    "api-version": "2022-12-01-preview",
+    "udid": "0522b4fe-0539-4c6c-ba70-e35ec865d8e2"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "udid": "0522b4fe-0539-4c6c-ba70-e35ec865d8e2",
+        "description": "Contoso Geofence GeoJSON",
+        "dataFormat": "geojson",
+        "connection": {
+          "type": "AzureBlobFile",
+          "msiClientId": "3263cad5-ed8b-4829-b72b-3d1ba556e373",
+          "linkedResource": "my-storage-account",
+          "scope": {
+            "containerName": "my-container",
+            "blobPath": "/my/blob/path2.json"
+          }
+        },
+        "downloadURL": "https://us.atlas.microsoft.com/mapData/0522b4fe-0539-4c6c-ba70-e35ec865d8e2",
```

Comment:
update downloadURL to `https://{geography}.atlas.microsoft.com/dataRegistries/{udid}/content?api-version=2022-12-01-preview`

Reply:
Fixed